module/core/src/Application/Request/ParamConverter/GridConfigurationParamConverter.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See license.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\Core\Application\Request\ParamConverter;
+
+use Ergonode\Grid\GridConfigurationInterface;
+use Ergonode\Grid\RequestGridConfiguration;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ */
+class GridConfigurationParamConverter implements ParamConverterInterface
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function apply(Request $request, ParamConverter $configuration): void
+    {
+        $requestGridConfiguration = new RequestGridConfiguration($request);
+        $request->attributes->set($configuration->getName(), $requestGridConfiguration);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function supports(ParamConverter $configuration): bool
+    {
+        return GridConfigurationInterface::class === $configuration->getClass();
```

Comment:
I think hire shoudl be specific class not interface

Reply:
If we use specific class it will be hard to create custom GridConfiguration object. Are you sure?

Reply:
Interface is for possibility to create diffrent object that create configuration from given data.
In this param converter have line:
```
return $requestGridConfiguration = new RequestGridConfiguration($request); 
```
thats mean the converter return specific class RequestGridConfiguration
if you create another converter for other class base on this interface , it be possibility to go erros becous of  use not correct converter